### ISSUE TITLE
Dashboard

### ISSUE THREAD

**Issue by GeorgiChochov** [AbsaOSS/enceladus#103]
We should have a dashboard view as the welcome page when a user logs in. This should show:
- latest runs for the user's datasets
- statistics aggregated from previous runs
- ???

**Comment by Zejnilovic** [comment]
If the runs and datasets will be user based, so I don't see what you are running and vice versa. We might have an export/import feature to be able to share schemas, mapping tables, etc.

**Comment by GeorgiChochov** [comment]
> If the runs and datasets will be user based, so I don't see what you are running and vice versa. We might have an export/import feature to be able to share schemas, mapping tables, etc.

Yes! We have #30 to restrict access and visibility and #34 to handle export/import. I appreciate getting suggestions!

**Comment by Zejnilovic** [comment]
> > If the runs and datasets will be user based, so I don't see what you are running and vice versa. We might have an export/import feature to be able to share schemas, mapping tables, etc.
> 
> Yes! We have #30 to restrict access and visibility and #34 to handle export/import. I appreciate getting suggestions!

My suggestion stands easy/fast button for import on the dashboard, maybe even export. 
Maybe we get a UX designer 😄
